jaxrs-clients/src/main/java/feign/TextDelegateDecoder.java

```diff
@@ -48,7 +48,7 @@ public Object decode(Response response, Type type) throws IOException, FeignExce
             contentTypes = ImmutableSet.of();
         }
         // In the case of multiple content types, or an unknown content type, we'll use the delegate instead.
-        if (contentTypes.size() == 1 && Iterables.getOnlyElement(contentTypes, "").equals(MediaType.TEXT_PLAIN)) {
+        if (contentTypes.size() == 1 && Iterables.getOnlyElement(contentTypes, "").contains(MediaType.TEXT_PLAIN)) {
```

Comment:
Let me go re-read the spec.  I originally had that, but then changed to `contains`.